src/main/java/org/example/lesson33/lesson30interface/Main.java

```diff
@@ -0,0 +1,55 @@
+package org.example.lesson33.lesson30interface;
+
+import org.example.lesson33.lesson30interface.Stackable;
+
+import java.util.SimpleTimeZone;
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {
+        Stackable stack = new Stack(5);
+
+        try {
+            stack.addElementToStack(10);
+        } catch (StackIsFullException e) {
+            System.out.println(e.getMessage());
+        }
+        try {
+            stack.addElementToStack(20);
+        } catch (StackIsFullException e) {
+            System.out.println(e.getMessage());
+        }
+        try {
+            stack.addElementToStack(30);
+        } catch (StackIsFullException e) {
+            System.out.println(e.getMessage());
+        }
```

Comment:
Ви можете обгорнути декілька рядків коду в try-catch
```suggestion
        try {
            stack.addElementToStack(10);
            stack.addElementToStack(20);
            stack.addElementToStack(30);
        } catch (StackIsFullException e) {
            System.out.println(e.getMessage());
        }
```